tests/ssg_test_suite/rule.py

```diff
@@ -187,13 +186,13 @@ def _final_scan_went_ok(self, runner, rule_id):
             logging.error(msg)
         return success
 
-    def _matches_target(self, rule_dir, targets):
```

Comment:
Nice naming improvement.

Reply:
I don't like when you call a rule a target because it is confused with a target system.